app/helpers/review_mapping_helper.rb

```diff
@@ -2,7 +2,175 @@ module ReviewMappingHelper
   def create_report_table_header(headers = {})
     render partial: 'report_table_header', locals: { headers: headers }
   end
+#ADDED: Get participant details if user is added to the assignment
+#RENAMED: get_participant_or_reviewer -> find_assignment_participant
+  # @param user_id [Integer] The ID of the user.
+  # @param parent_id [Integer] The ID of the assignment.
+  # @return [AssignmentParticipant or nil] The participant record if found, or nil if not found.
+
+# Reasoning:
+  #  - Reflects that this method locates an AssignmentParticipant for a given user and assignment.
+  #  - Removes ambiguity around "or reviewer," since it specifically returns a participant.
+  def find_assignment_participant(user_id, parent_id)
```

Comment:
Why not just remove this method and call the db query wherever it's needed?  That would seem to be a much clearer way to code.